### ISSUE TITLE
Unable to use `gRPC proto files` importer type on macOS

### ISSUE THREAD

**Issue by agoodsell5** [riok/Kreya#29]
**Describe the bug**
Unable to use a local file based importer on macOS.

I am unable to use the file import method on macOS. It works as expected with the same proto file on Windows.

**To Reproduce**
1. Go to Importers
2. Click on Type
3. Scroll down to gRPC proto files
4. Tick 'Create an operation for each API method`
5. Click 'Proto files' tab
6. Click 'Add proto files`
7. Select file
8. Click 'Save'
9. See error

**Expected behavior**
Expect this to import the service as it does on Windows

**Screenshots**
Here is a copy of the proto file, it is just a Greeter example:

```
syntax = "proto3";

option csharp_namespace = "Test.Namespace";

package greet;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply);
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings.
message HelloReply {
  string message = 1;
}
```

**Environment (if possible, copy the information from the error dialog or the About menu):**
```
{
  "kreyaVersion": "1.3.1",
  "platform": "MacIntel",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_16) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}

System.InvalidCastException: Unable to cast object of type 'Kreya.Data.Models.ImportStream`1[Kreya.Grpc.Data.Models.Importer.GrpcServerReflectionImporterOptions]' to type 'Kreya.Data.Api.Models.IImportStream`1[Kreya.Grpc.Data.Models.Importer.ProtoFileImporterOptions]'.
   at Kreya.Data.Api.StorageHooks.ImportStreamStorageHookAdapter`1.PrepareImportStreamForSave(IImportStream importStream)
   at Kreya.Data.ProjectStorage.<>c.<StoreProject>b__5_0(IImportStreamStorageHookAdapter adapter, ImportStream importStream)
   at Kreya.Data.ProjectStorage.CallImportStreamHooks(Project project, Func`3 hook)
   at Kreya.Data.ProjectStorage.StoreProject(Project proj)
   at Kreya.Core.Importer.ImporterService.CreateImportStream(ImportStream importStream)
   at Kreya.UI.Importer.ImporterUiService.CreateImportStream(CreateImportStreamModel createModel)
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

**Comment by CommonGuy** [comment]
Strange, creating an importer on macOS should work the same as on windows. Since we're unable to reproduce this error on windows and macOS, could you provide the following information?
1. Is this the only importer in the project?  Or do other importers exist?
2. Does creating a server reflection importer work? You could try it with the `https://grpcb.in:9001` URL, untick "Create an operation for each API method" if you don't want to have a bunch of generated operations.
3. Does creating a proto files importer work, if you don't select any files?

**Comment by agoodsell5** [comment]
1. There was another importer in the project, which was a server reflection importer for the same service, this worked perfectly. I have just tried creating a new project that does not contain any importers, and this did allow me to successfully use the proto files importer.
2. As stated in 1, yes server reflection works perfectly
3. No this did not work either, it gave the same error

So it would seem this is actually a problem that I cannot add a second importer to a project. If I first create a proto file importer this works, but then trying to add a second importer of any type does not.

**Comment by latonz** [comment]
We just released Kreya 1.4.0-beta.2 which should include this bugfix: https://kreya.app/release-notes/beta. You can switch to the Beta release channel in the app via Kreya -> About. Change the release channel to Beta and search for updates. It should then install the newest Beta version. Please note that the Beta channel may contain some unstable features.

**Comment by latonz** [comment]
BugFix is released with Kreya 1.4.

**Comment by agoodsell5** [comment]
Just to say this absolutely fixed the issue! Thank you very much for the speedy fix!
